src/common/tse_internal.h

```diff
@@ -33,8 +33,8 @@
  * Author: Di Wang  <di.wang@intel.com>
  */
 
-/* NB: tse_task_private is TSE_PRIV_SIZE = 1016 bytes for now */
-#define TSE_TASK_ARG_LEN		888
+/* NB: tse_task_private is TSE_PRIV_SIZE = 1528 bytes for now */
+#define TSE_TASK_ARG_LEN		1400
```

Comment:
There is a comment in task_internal.h that appears to have the wrong size...maybe we should update remove the numeric size

Reply:
Hmm, is TSE_TASK_ARG_LEN related to TSE_PRIV_SIZE ?  Can we define one in terms of the other?   Additionally, can we remove the 1528 here as one can just go lookup the value in the other header?